rm_setup_and_git_commands.md

```diff
@@ -0,0 +1,79 @@
+# Git Commands and Setup
+
+This file is to document environmental setup and git commands
+
+## Setup Dev Environment
+
+### Install needed applications
+
+Ensure that you have docker and docker-compose installed on your machine, as well as the rest of the Python stack.
+ * GitBash
+ * Python 3.6.7

```

Comment:
Should we also include node here?

Reply:
I think node is technically pulled in by Docker: we have 9.7 on our machines, but are using 10.x in Docker.